### DISCUSSION TITLE
Error when trying to create new account: Invalid email, are you sure the email is correct?

### DISCUSSION THREAD

**Discussion by devtud** [simple-login/app#1617] [Self-hosting question] [answered]
Heya

I'm stuck just after I deployed the app and hopefully someone could help me to get through this.

So when I signup, a toast in browser shows: "Invalid email, are you sure the email is correct?".

No error logs in simplelogin container, so I didn't figure out why the error message in the browser. The user was created in db and the email was triggered, but it doesn't seem to be delivered.

```
2023-03-04 01:36:58,716 - SL - DEBUG - "/code/app/email_utils.py:250" - send_email() -  - send email to ******@yahoo.com, subject Just one more step to join SimpleLogin
```

In postfix logs I can see:
```
Mar 04 01:38:54 app postfix/smtpd[719]: lost connection after CONNECT from unknown[10.42.0.23]
Mar 04 01:38:54 app postfix/smtpd[719]: disconnect from unknown[10.42.0.23] commands=0/0
```

10.42.0.23 is the ip of the simplelogin container, but not sure "unknown" is ok there.

All app is deployed in a Kubernetes cluster and Postfix runs also inside a container. I didn't expose the port 25 to the outside world, but that shouldn't impact the comunication between simplelogin and Postfix I guess.

Any idea?

I made postfix more verbose and caught this one, but I'm not sure it's a problem. I'm referring to `domain = ` which is empty. Is there any issue with that?
```
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_str: /etc/postfix/pgsql-relay-domains.cf: user = simplelogin
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_str: /etc/postfix/pgsql-relay-domains.cf: password = *******
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_str: /etc/postfix/pgsql-relay-domains.cf: dbname = simplelogin
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_str: /etc/postfix/pgsql-relay-domains.cf: result_format = %s
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_int: /etc/postfix/pgsql-relay-domains.cf: expansion_limit = 0
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_str: /etc/postfix/pgsql-relay-domains.cf: query = SELECT domain FROM custom_domain WHERE domain='%s' AND verified=true    UNION SELECT '%s' WHERE '%s' = '*****.com' LIMIT 1;
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_str: /etc/postfix/pgsql-relay-domains.cf: domain =
Mar 04 01:38:54 app postfix/smtpd[719]: cfg_get_str: /etc/postfix/pgsql-relay-domains.cf: hosts = dev-pg-postgresql
```

**Comment by devtud** [comment, marked as answer]
I added some logs and figured out that there was the dkim library issue. upgrade to 1.1.0 and worked.

**Reply by tayeizana** [reply]
How can one go about upgrading to 1.1.0? And are you upgrading the dkim library?

**Reply by FallingReign** [reply]
I have this issue too. So would also like to know. 

Been struggling for days so if you’re using a compose file I’d love to see it.
